qadence/types.py

```diff
@@ -122,6 +124,26 @@ class LTSOrder(StrEnum):
     """ST4."""
 
 
+class BasisSet(StrEnum):
```

Comment:
I think `BasisSet` is too generic in this context, IMHO.

Reply:
What about `BasisFM` (not to be too verbose) ?

Reply:
Could also do `ScalingFM` instead of `ReuploadScaling`

Reply:
> Could also do `ScalingFM` instead of `ReuploadScaling`

I would definitely rename this one tbh.